pennylane/transforms/defer_measurements.py

```diff
@@ -101,9 +105,294 @@ def null_postprocessing(results):
+            processing_fn = lambda x: eqn.primitive.bind(x, other, **eqn.params)
+            return meas_val._apply(processing_fn)
+
+        def eval(self, jaxpr: "jax.core.Jaxpr", consts: list, *args) -> list:
+            """Evaluate a jaxpr.
+
+            Args:
+                jaxpr (jax.core.Jaxpr): the jaxpr to evaluate
+                consts (list[TensorLike]): the constant variables for the jaxpr
+                *args (tuple[TensorLike]): The arguments for the jaxpr.
+
+            Returns:
+                list[TensorLike]: the results of the execution.
+
+            """
+            self._env = {}
+            self.setup()
+
+            for arg, invar in zip(args, jaxpr.invars, strict=True):
+                self._env[invar] = arg
+            for const, constvar in zip(consts, jaxpr.constvars, strict=True):
+                self._env[constvar] = const
+
+            for eqn in jaxpr.eqns:
+
+                custom_handler = self._primitive_registrations.get(eqn.primitive, None)
+                if custom_handler:
+                    invals = [self.read(invar) for invar in eqn.invars]
+                    outvals = custom_handler(self, *invals, **eqn.params)
+                elif isinstance(eqn.outvars[0].aval, AbstractOperator):
+                    outvals = self.interpret_operation_eqn(eqn)
+                elif isinstance(eqn.outvars[0].aval, AbstractMeasurement):
+                    outvals = self.interpret_measurement_eqn(eqn)
+                else:
+                    invals = [self.read(invar) for invar in eqn.invars]
+                    if any(isinstance(inval, MeasurementValue) for inval in invals):
+                        outvals = self.resolve_mcm_values(eqn, invals)
+                    else:
+                        outvals = eqn.primitive.bind(*invals, **eqn.params)
+
+                if not eqn.primitive.multiple_results:
+                    outvals = [outvals]
+                for outvar, outval in zip(eqn.outvars, outvals, strict=True):
+                    self._env[outvar] = outval
+
+            # Read the final result of the Jaxpr from the environment
+            outvals = []
+            for var in jaxpr.outvars:
+                outval = self.read(var)
+                if isinstance(outval, qml.operation.Operator):
+                    outvals.append(self.interpret_operation(outval))
+                else:
+                    outvals.append(outval)
+            self.cleanup()
+            self._env = {}
+            return outvals
+
+    @DeferMeasurementsInterpreter.register_primitive(measure_prim)
+    def _(self, wires, reset=False, postselect=None):
+        if self.state["cur_idx"] >= len(self._aux_wires):
+            raise ValueError(
+                "Not enough auxiliary wires provided to apply specified number of mid-circuit "
+                "measurements using qml.defer_measurements."
+            )
+
+        with qml.QueuingManager.stop_recording():
```

Comment:
What's stop recording needed for? Potentially we may need to cut queuing out of some program capture code if this is needed.

Reply:
I put it there "just in case". Nothing should be using queuing in the capture enabled pipeline, so it _should_ be fine to remove it

Reply:
Removed locally.